src/app/board/components/header/header.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-board-page-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss'],
+})
+export class HeaderComponent {
+  @Input() boardTitle: string | undefined = '';
```

Comment:
Why undefined for each @Input? in what cases could it be undefined?

Reply:
> Why undefined for each @input? in what cases could it be undefined?

Done